ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.1.yaml

```diff
@@ -0,0 +1,192 @@
+base_images:
+  ansible:
+    name: "4.16"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.16"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.16"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.15"
+  tools:
+    name: "4.16"
+    namespace: ocp
+    tag: tools
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+releases:
+  initial:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.1"
+  intermediate42:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.2"
+  intermediate43:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.3"
+  intermediate44:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.4"
+  intermediate45:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.5"
+  intermediate46:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.6"
+  intermediate47:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.7"
+  intermediate48:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.8"
+  intermediate49:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.9"
+  intermediate410:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.10"
+  intermediate411:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.11"
+  intermediate412:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.12"
+  intermediate413:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.13"
+  intermediate414:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.14"
+  intermediate415:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.15"
+  latest:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.14"
```

Comment:
It's not used in below job, right?

Reply:
Copied from 4.15, latest:release is required and in 4.15 its value is 4.13

Reply:
Copied from 4.15, latest:release is required and in 4.15 its value is 4.13